docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,26 @@
 # Staging release notes
 
+## April 2024
+
+**11 April 2024**
+
+Frontend version: 1.0.0-20240328.0943 | Backend version: 1.97.2-20240405.1708
+
+
+| Type | Change | Description |
+| ---- | ------ | ----------- |
+| **Feature** | SAML certificates expiry notifications | The `notify-expiring-saml-certs` job has been enhanced to email tenant admins about SAML certificate expirations. Notifications will exclude certificates for applications not aligned with TechPass standards. |
```

Comment:
Dun need to mention the job name (it's unknown to the tenants) & remove the 'Notifications will exclude...' sentence

I think sth like 'Tenant admins will receive email notifications when an active SAML app certificate is nearing expiration' will do.